src/frontend/apps/standalone_site/src/features/ClassRoom/components/Update/ClassRoomUpdate.tsx

```diff
@@ -82,6 +82,7 @@ function ClassRoomUpdate() {
       can_access_dashboard: true,
       can_update: true,
     },
+    isFromWebsite: true,
```

Comment:
in the body of commit `f3570489e39edffb5604d6b03421bb8a62fdf695`

`s/- only if comming from website/- only if coming from website`